### PR TITLE
New landing pages on tutorials and guides

### PR DESCRIPTION
**Pull request by lindtdev**
This PR adds new texts that have been written by Carolin for our landing pages on Guides and Tutorials&Howtos.

### CODE REVIEW

**Review by Carolin519 — APPROVED** [review]

File: content/kubermatic/master/tutorials_howtos/_index.en.md [hunk 1]
```diff
@@ -5,8 +5,12 @@ weight = 20
 +++
 
 
-This section contains tutorials that will show you how to execute basic tasks with your KKP cluster.
+Are you just embarking on your cloud adoption journey but feeling a bit overwhelmed on how to get started? Then you’re in the right place here:)
```
Comment (moadqassem) [inline content/kubermatic/master/tutorials_howtos/_index.en.md:8]: I am not sure, but is it really a good idea to have emojis in the documentation?

Comment (Carolin519) [reply content/kubermatic/master/tutorials_howtos/_index.en.md]: Removed it.

File: content/kubermatic/v2.17/tutorials_howtos/_index.en.md [hunk 2]
```diff
@@ -4,9 +4,12 @@ date = 2020-02-10T11:07:15+02:00
 weight = 20
 +++
 
+Are you just embarking on your cloud adoption journey but feeling a bit overwhelmed on how to get started? Then you’re in the right place here:)
```
Comment (moadqassem) [inline content/kubermatic/v2.17/tutorials_howtos/_index.en.md:7]: Unfortunately the emoji here left behind

**Review by moadqassem — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**